test/algorithms_test.dart

```diff
@@ -389,10 +390,9 @@ class OC implements Comparable<OC> {
   String toString() => 'OC[$id,$order]';
 }
 
-int ocId(OC oc) => oc.id;
-int ocOrder(OC oc) => oc.order;
+int _ocOrder(OC oc) => oc.order;
 
-int compareInt(int a, int b) => a - b;
+int _compareInt(int a, int b) => a - b;
```

Comment:
(Test files should *not* have to care whether their functions are public or not. They likely shouldn't be. But if it's just for consistency, that's fine.)

Reply:
I like consistency. It makes parsing the file easier. Is this used outside the library or not, etc...